x-pack/packages/security-solution/ecs_data_quality_dashboard/impl/data_quality/data_quality_panel/index_properties/helpers.test.ts

```diff
@@ -5,47 +5,32 @@
  * 2.0.
  */
 
-import { EcsFlat } from '@elastic/ecs';
-
 import {
   getMappingsProperties,
   getSortedPartitionedFieldMetadata,
   hasAllDataFetchingCompleted,
 } from './helpers';
 import { mockIndicesGetMappingIndexMappingRecords } from '../../mock/indices_get_mapping_index_mapping_record/mock_indices_get_mapping_index_mapping_record';
 import { mockMappingsProperties } from '../../mock/mappings_properties/mock_mappings_properties';
-import { EcsMetadata } from '../../types';
-
-const ecsMetadata: Record<string, EcsMetadata> = EcsFlat as unknown as Record<string, EcsMetadata>;
+import { EcsFlatTyped } from '../../constants';
 
 describe('helpers', () => {
   describe('getSortedPartitionedFieldMetadata', () => {
     test('it returns null when mappings are loading', () => {
       expect(
         getSortedPartitionedFieldMetadata({
-          ecsMetadata,
+          ecsMetadata: EcsFlatTyped,
           loadingMappings: true, // <--
           mappingsProperties: mockMappingsProperties,
           unallowedValues: {},
         })
       ).toBeNull();
     });
 
-    test('it returns null when `ecsMetadata` is null', () => {
-      expect(
-        getSortedPartitionedFieldMetadata({
-          ecsMetadata: null, // <--
-          loadingMappings: false,
-          mappingsProperties: mockMappingsProperties,
-          unallowedValues: {},
-        })
-      ).toBeNull();
-    });
-
```

Comment:
For reviewers:

This test was removed because it tests an impossible state. Details:
https://github.com/elastic/kibana/pull/185025/files#r1631679640